config/globals.js

```diff
@@ -38,6 +38,8 @@ const globals = {
 	BODYPARSER_JSON_LIMIT: process.env.BODYPARSER_JSON_LIMIT || '20mb',
 	DATABASE_AUDIT: process.env.DATABASE_AUDIT || 'false',
 	DB_URL: process.env.DB_URL || defaultDbUrl,
+	DB_USERNAME: process.env.DB_USERNAME,
```

Comment:
no default lies for the DB Name?

Reply:
nope, these are credentials... the globals seems to be a pre-step next turn will move everything to the config